backend/app/services/dingtalk/docs_service.py

```
@@ -0,0 +1,450 @@
+            modified_time = result.get("modified_time", datetime.now().isoformat())
+
+            return {
+                "doc_id": doc_id,
+                "title": title,
+                "modified_time": modified_time,
+                "modified_time_formatted": self._format_modified_time(modified_time),
+                "content_type": result.get("content_type", "markdown"),
+                "url": doc_url,
+            }
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)